code/modules/mining/alloys.dm

```diff
@@ -0,0 +1,23 @@
+//Alloys that contain subsets of each other's ingredients must be ordered in the desired sequence
+
+/datum/alloy
```

Comment:
We already have smelting_recipe datums.

I dislike this because you have presumably made smelting a single ore just spit out its sheettype, which loses some functionality, and also because it doesn't use material IDs like smelting already does.